docs/docusaurus-authoring-guide.mdx

```diff
@@ -0,0 +1,202 @@
+---
+id: authoring-guide
+title: Authoring Guide
+sidebar_label: Authoring Guide
+---
+
+You can write content using [GitHub-flavored Markdown syntax](https://github.github.com/gfm/).
+
+## Markdown Syntax
+
+To serve as an example page when styling markdown based Docusaurus sites.
+
+## Headers
+
+# H1 - Create the best documentation
+
+## H2 - Create the best documentation
+
+### H3 - Create the best documentation
+
+#### H4 - Create the best documentation
+
+##### H5 - Create the best documentation
+
+###### H6 - Create the best documentation
+
+---
+
+## Emphasis
+
+Emphasis, aka italics, with *asterisks* or _underscores_.
+
+Strong emphasis, aka bold, with **asterisks** or __underscores__.
+
+Combined emphasis with **asterisks and _underscores_**.
+
+Strikethrough uses two tildes. ~~Scratch this.~~
+
+---
+
+## Lists
+
+1. First ordered list item
+1. Another item
+   - Unordered sub-list.
+1. Actual numbers don't matter, just that it's a number
+   1. Ordered sub-list
+1. And another item.
+
+* Unordered list can use asterisks
```

Comment:
For our purposes, I'd like to pick one unorder list style and stick with it so all our docs are consistent.

Reply:
I prefer * over - if we are picking

Reply:
This document was one of the ones that gets installed with Docusaurus out of the box, so I honestly didn't even really look at it that closely, but I'm happy to make any changes that we want to make to it. 👍🏼